### PR TITLE
desk bell from SS13 (tgstation)

### PR DESCRIPTION
**Pull request by DocNITE**
## About the PR
Add desk bell, what can be used on stands/tables for calling anyone.

**Media**
image

### CODE REVIEW

File: Resources/Audio/Items/desk_bell_ring.ogg
Comment (AJCM-git) [inline Resources/Audio/Items/desk_bell_ring.ogg:1]: This needs its copyright info added to attributions.yml in this folder

File: Resources/Prototypes/Entities/Objects/Misc/desk_bell.yml
Comment (AJCM-git) [inline Resources/Prototypes/Entities/Objects/Misc/desk_bell.yml:1]: make it anchorable

File: Resources/Textures/Objects/Misc/desk_bell.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/e1142f20f5e4661cb6845cfcf2dd69f864d67432",
```
Comment (AJCM-git) [inline Resources/Textures/Objects/Misc/desk_bell.rsi/meta.json:4]: try linking the DMI file you took this from *at* this commit

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Misc/desk_bell.yml
Comment (moonheart08) [reply Resources/Prototypes/Entities/Objects/Misc/desk_bell.yml:1]: not needed.

File: Resources/Audio/Items/desk_bell_ring.ogg
Comment (DocNITE) [reply Resources/Audio/Items/desk_bell_ring.ogg:1]: What is attributions.yml?

**Comment by DocNITE** [comment]
Okay, i added less copyright links

File: Resources/Textures/Objects/Misc/desk_bell.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/blob/master/icons/obj/bureaucracy.dmi",
```
Comment (AJCM-git) [inline Resources/Textures/Objects/Misc/desk_bell.rsi/meta.json:4]: you need to link the file *at* the commit (copy permalink)
image

File: Resources/Audio/Items/attributions.yml [hunk 3]
```diff
@@ -27,3 +27,8 @@
   license: "CC-BY-SA-3.0"
   copyright: "Taken from tgstation, modified by brainfood1183 (github) for ss14"
   source: "https://github.com/tgstation/tgstation/blob/529d97cb1c105bcd548e95a9c9070bbf5253dd81/sound/items/AirHorn.ogg"
+
+- files: ["desk_bell_ring.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Taken from tgstation"
+  source: "https://github.com/tgstation/tgstation/blob/master/sound/machines/microwave/microwave-end.ogg"
```
Comment (AJCM-git) [inline Resources/Audio/Items/attributions.yml:34]: same
Also this sound was added to to TG in this PR https://github.com/tgstation/tgstation/pull/32500 the author seems to still be active so please reach out to them and ask them if 1) They **made** that sound 2) They agree with this license . If both are true you need to make the copyright "Made by *author name* for tgstation"

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Resources/Audio/Items/attributions.yml (hunk 3 above)
Comment (DocNITE) [reply Resources/Audio/Items/attributions.yml]: Okay, done (x2)

Comment (AJCM-git) [reply Resources/Audio/Items/attributions.yml]: well, what did they say? can you post an screenshot or get them to comment here

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Resources/Audio/Items/desk_bell_ring.ogg
[the remark this answers is not in the document]
Comment (DocNITE) [reply Resources/Audio/Items/desk_bell_ring.ogg:1]: Mono? What is it?

**Comment by DocNITE** [comment]
Done (x3)

**Pull request merged**